app/models/project_user.rb

```diff
@@ -0,0 +1,3 @@
+class ProjectUser < ApplicationRecord
+  belongs_to :project
```

Comment:
association to users?

Reply:
user model is not created yet... so thats why i did not add it here